docs/mkdocs.yml

```
@@ -1,5 +1,5 @@
 site_name: react-jsonschema-form documentation
-docs_dir: docs
+docs_dir: src
```

```suggestion
docs_dir: docs/src
```

Just curious, why did you prefer using `docs/src` instead of `docs`?

---

From [mkdocs documentation](https://www.mkdocs.org/user-guide/configuration/#docs_dir) it can be relative or absolute, so in this case it should be correct.